test/unit/org/apache/cassandra/index/internal/CassandraIndexTest.java

```diff
@@ -558,9 +561,17 @@ public void indexStatementsWithConditions() throws Throwable
     @Test
     public void indexCorrectlyMarkedAsBuildAndRemoved() throws Throwable
     {
-        String selectBuiltIndexesQuery = String.format("SELECT * FROM %s.\"%s\"",
+        String selectBuiltIndexesQuery = String.format("SELECT table_name, index_name FROM %s.\"%s\"",
```

Comment:
I'd say that checking that the index build doesn't write the legacy `IndexInfo.value` column had some value, but I'm not against the simplification if you prefer it this way.

Reply:
Nah I see your point. +1.